radix-engine/src/model/global/substates.rs

```diff
@@ -8,8 +8,7 @@ pub enum GlobalAddressSubstate {
     Component(ComponentId),
     Resource(ResourceManagerId),
     Package(PackageId),
-    // We may consider using another enum here so we can map system addresses to nodes of different type.
-    System(EpochManagerId),
+    System(SystemId),
```

Comment:
Can we just separate `System` into `Clock` and `EpochManager`. I think may be simpler if these were more flattened out?

Reply:
yup, sure